server/sonar-web/src/main/js/main/nav/global/search-view.js

```diff
@@ -245,7 +245,7 @@ export default Marionette.LayoutView.extend({
   getGlobalDashboardFindings (q) {
     const dashboards = this.model.get('globalDashboards') || [];
     const items = dashboards.map(function (d) {
-      return { name: d.name, url: '/dashboard/index?did=' + encodeURIComponent(d.key) };
+      return { name: d.name, url: window.baseUrl + '/dashboard/index?did=' + encodeURIComponent(d.key) };
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)